app/_how-tos/adjust-header-names-in-request.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Adjust header names in a request
+content_type: how_to
+
+description: Change the names of headers sent in a request using the Post-Function plugin.
+
+products:
+    - gateway
+
+works_on:
+    - on-prem
+    - konnect
+
+tools:
+    - deck
+
+prereqs:
+  entities:
+    services:
+        - example-service
+    routes:
+        - example-route
+
+min_version:
+  gateway: '3.4'
+
+plugins:
+  - post-function
+  - rate-limiting
+
+entities:
+  - service
+  - route
+  - plugin
+
+tags:
+  - headers
+  - serverless
+  - rate-limiting
+
+tldr:
+  q: How do I adjust request header names from multiple sources?
+  a: |
+   You can use the serverless Post-Function plugin to detect headers in a request and transform them into custom header names.
+
+   In this tutorial, we'll edit two types of headers: headers set by a plugin (in this case, Rate Limiting), and latency headers from {{site.base_gateway}}.
+   
+   In this tutorial, we'll enable the Post-Function plugin in the `header_filter` phase, where it will look for a configured list of headers, then transform those headers into different names.
```

Comment:
```suggestion
   We'll enable the Post-Function plugin in the `header_filter` phase, where it will look for a configured list of headers, then transform those headers into different names.
```